### PR TITLE
deleted last_modified function & datetime package in models.py 

### PR DESCRIPTION
**Pull request by mkshah605**
Deleted both the last_modified function and the datetime package in the models.py file, as neither of them are being used. This change successfully passed nox testing.

### CODE REVIEW

File: planet/models.py [hunk 1]
```diff
@@ -157,16 +156,6 @@ def num_bytes_downloaded(self):
         '''
         return self.response.num_bytes_downloaded
 
-    def last_modified(self):
```
Comment (kevinlacaille) [inline planet/models.py:160]: I believe I was wrong. I think we should keep this function. Although this planet module doesn't call for this function, `last_modified()`, a user might actually want to use this function. So, I suggest putting it back in along with the `datetime` module.

Now, my first comment still stands - we need to either change `datetime.strptime()` to `datetime.datetime.strptime()`. Or you can do `from datetime import datetime`, then `datetime.strptime()` will work :)

File: planet/models.py [hunk 2]
```diff
@@ -13,7 +13,6 @@
 # limitations under the License.
 """Manage data for requests and responses."""
 import copy
-import datetime
```
Comment (kevinlacaille) [inline planet/models.py:16]: From my comment bellow, this should go back in either as is or as `from datetime import datetime` - chef's choice!

**Review by kevinlacaille — COMMENTED** [review]
I was wrong, I think we should keep this function in! See my comments for my reasoning 👍 

Now, I think we should add a test for this function in `tests/unit/test_models.py`!

File: planet/models.py [hunk 3]
```diff
@@ -151,15 +142,13 @@ def size(self):
...
         '''The number of bytes downloaded.
-
         :returns: number of bytes downloaded
         :rtype: int
         '''
         return self.response.num_bytes_downloaded
 
     def last_modified(self):
         '''Read the last-modified header as a datetime, if present.
-
```
Comment (jreiberkyle) [inline planet/models.py:162]: Is there a reason these empty lines were removed? Their presence increases readability of the docstrings. I suggest undoing all the 'remove empty line' changes

**Review by jreiberkyle — CHANGES_REQUESTED** [review]
The new tests are focused and clear and cover the expected variations. They are great. I suggest removing all of the unrelated changes, ie the removal of empty lines.

**Review by kevinlacaille — APPROVED** [review]
Looks good to me! 🎉

**Review by jreiberkyle — APPROVED** [review]
Looks great. Nicely done!

**Pull request merged**